src/components/meeting/preparing/MeetingOrganizerWrapper.js

```diff
@@ -23,6 +24,9 @@ export const MeetingOrganizerWrapper = ({ type : propType }) => {
         if(type === "organizer") {
             interviewAPI.getNewestInterview(uuid)
                 .then(data => {
+                    if (data?.application?.stage?.type === "HR_INTERVIEW") {
```

Comment:
to jest dość mocno RS-specific, ale widzę niżej że wołamy wprost callbacki z RSa, a nie jakies przekazywane, więc chyba nie ma co się o to rzucać xD